docs/sending-techniques/sending-data-cli-runner/sending-data-cli-runner.md

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_label: Overview
+---
+
+# Run Slack CLI commands in GitHub Actions workflows 
+
+This technique uses the Slack CLI in GitHub Actions to run commands through workflows.
+
+Setting up a CI/CD pipeline [requires](https://docs.slack.dev/tools/slack-cli/guides/authorizing-the-slack-cli/#ci-cd) authorization using a service token. [Service tokens](https://docs.slack.dev/tools/slack-cli/guides/authorizing-the-slack-cli/#obtain-token) are long-lived, non-rotatable user tokens that don’t expire.
+
+## Setup
+
+1. Add your service token as a [repository secret](https://docs.github.com/en/actions/how-tos/write-workflows/choose-what-workflows-do/use-secrets#creating-secrets-for-a-repository) called SLACK_SERVICE_TOKEN.
```

Comment:
📚 suggestion(non-blocking): Including more explicit steps to gather this, and what an expected value might resemble, can be useful to include!